snapcraft/internal/lxd.py

```diff
@@ -80,10 +81,14 @@ def _pull_file(self, src, dst):
         check_call(['lxc', 'file', 'pull',
                     '{}/{}'.format(self._container_name, src), dst])
 
-    def _container_run(self, cmd):
+    def _container_run(self, cmd, **kwargs):
         # Set HOME here as 'lxc config set ... environment.HOME' doesn't work
-        check_call(['lxc', 'exec', self._container_name, '--env',
-                   'HOME=/{}'.format(self._project_folder), '--'] + cmd)
+        # Use 'cd' because --env has no effect with sshfs mounts
```

Comment:
This doesn't actually set `$HOME` at all though, right? Does it need to be set? Or does the comment above need to be removed?